components/prism-naniscript.js

```diff
@@ -0,0 +1,172 @@
+(function (Prism) {
+
+	function expressionDef(isMultiline) {
+		return RegExp(/\{[^\r\n\[\]{}]*\}/, isMultiline ? 'm': '');
+	};
```

Comment:
The m flag doesn't affect the pattern at all, so let's just make this a variable instead of a function.

```suggestion
	var expressionDef = /\{[^\r\n\[\]{}]*\}/;
```